paddle/framework/op_registry.h

```diff
@@ -307,22 +307,38 @@ class OpRegistry {
   }
 };
 
+class Registrar {
+ public:
+  void Touch() {}
+};
+
 template <typename OpType, typename ProtoMakerType>
-class OpRegisterHelper {
+class OpRegistrar : public Registrar {
```

Comment:
In google code style:
"Use a struct only for passive objects that carry data; everything else is a class."

https://google.github.io/styleguide/cppguide.html#Structs_vs._Classes